fusor-ember-cli/app/controllers/register-nodes.js

```diff
@@ -645,14 +664,19 @@ export default Ember.Controller.extend(ProgressBarMixin, NeedsDeploymentMixin, {
   updateAutoDetectedNodes(hostArray) {
     let autoDetectedNodesMultiMac = [];
     let autoDetectedNodesSingleMac = [];
+    let usedMacs = this.getPortMacAddresses();
 
-    hostArray.forEach((hostHash) => {
+    hostArray.forEach(hostHash => {
```

Comment:
I pretty much always prefer to omitting parentheses  because I feel like it's pretty clear whenever I see the fat arrow, but I can go either way in this case since we're using a block syntax.  If we're omitting the block syntax I definitely prefer to omit the params when possible.  We need to get that jscs style guide thing going.

```javascript
x => x + x;
```